st2common/st2common/service_setup.py

```diff
@@ -126,6 +127,9 @@ def setup(service, config, setup_db=True, register_mq_exchanges=True,
     if run_migrations:
         run_all_rbac_migrations()
 
+    if cfg.CONF.metrics.enable:
+        metrics_initialize()
```

Comment:
If we add a noop metrics driver and make that the default, do we still need this metrics enable config option?